doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3519,14 +3518,6 @@ Jobs not received after reset
 
 .. rst-class:: v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0
 
-NCSDK-24305: fota_download library sends FOTA_DOWNLOAD_EVT_FINISHED when unable to connect
-  The :ref:`lib_download_client` library does not resume a download if the device cannot connect to a target server.
-  This causes the :ref:`lib_fota_download` library to incorrectly assume that the download has completed.
-
-  **Workaround:** Set the :kconfig:option:`CONFIG_FOTA_SOCKET_RETRIES` Kconfig option to ``0``.
-
-.. rst-class:: v1-1-0
-
```

Comment:
We don't delete the Known issues since those exist in the older releases. The changes w.r.t this PR on the KI page is to replace  :ref:lib_download_client to  :ref:lib_downloader on line nos: 3501, 3502, and 3516.